RefrigeratorExe/RefrigeratorExe/Refrigerator.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace RefrigeratorExe
+{
+    internal class Refrigerator :IComparable<Refrigerator>
+    {
+        private string _model;
+
+
+        public Guid id { get; }
+        public string Model
+        {
+            get
+            {
+                return _model;
+            }
+            private set
+            {
+                if (!value.Equals(""))
+                    _model = value;
+                else
+                    throw new ArithmeticException("Illegal model ,model must contain a String ");
```

Comment:
1. Model is a string so the exception type is not arithmetic. ArgumentException is a good choice
2. Exception message should indicate the error and should not have spaces at the end of message: "Model cannot be empty"
3. Validate the Model length is reasonable by adding a const value that symbolizes the maximum length allowed.
private const int MaxModelLength = 50;
4. see my comment below on the constructor